Sources/TuistCloud/Services/CacheExistsService.swift

```diff
@@ -0,0 +1,76 @@
+import Foundation
+import TuistSupport
+
+public protocol CacheExistsServicing {
```

Comment:
Why splitting in 4 services rather than just having a single CacheService?

Reply:
My guess is that he weighed more the clarity and scoping of having one class per operation at the cost of introducing some duplications across services, over having a service that might grow into a massive class with many responsibilities. Both have pros and cons. I'd suggest that we merge and iterate if the drawbacks of his decision prove to have a very negative impact.

Reply:
The services are thin wrappers over the code generated by OpenAPI. The endpoints are not grouped, so it's better not to couple the services as well. I created a service per endpoint across all the other Tuist Cloud endpoints.